src/LayerEditor/ui/panels/regions_panel.py

```diff
@@ -275,41 +270,29 @@ def __init__(self, layer_heads, i_tab, parent):
         # === Data
         self.layer_heads = layer_heads
         # Reference model for region panel
-        self._i_tab = i_tab
-        # Set index of tab in the toolbox (used for callback to set header).
-        self.layer_id = None
-        # ID of the layer to which this Tab is related.
+        self.layer = layer
+        # Layer to which this Tab is related.
         self._layer_name = None
         # Name of the layer.
-        self._combo_id_to_idx = {0: 0}
+        self._combo_id_to_idx = {Region.none.id: 0}
```

Comment:
All tabs should have the same region combo box, so ideally we should create that only once and pass it to the LayerTabs.

Reply:
I figured that it is simpler to keep a list of regions in RegionsPanel which will correspond to combo box items. `_combo_id_to_idx` is therefore eliminated. Done.

Reply:
`_combo_id_to_idx` in `RegionPanleTab` were replaced by one list of regions in `RegionsPanel`, which corresponds to items in ComboBox.

Reply:
https://github.com/GeoMop/GeoMop/blob/df105ad9c7689fa3f2b141a522ec791c1d7f7df6/src/LayerEditor/ui/panels/regions_panel.py#L31-L33